### PR TITLE
Fix issue with jquery init

### LINKED ISSUE #4: Need help with usage
I'm trying to incorporate this package in a project, but it's not preventing two people from editing the same item simultaneously. I suspect I'm not using the AdminLockingMixin correctly.

I have admin_locking installed:

```
INSTALLED_APPS = [
    'users.apps.UsersConfig', # custom user model
    'items.apps.ItemsConfig',
    'plans.apps.PlansConfig',
    'grappelli',
    'admin_locking',
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
]
```
Admin looks like this:

```
class ItemAdmin(admin_locking.AdminLockingMixin, admin.ModelAdmin):
\...
admin.site.register(Item, ItemAdmin)
```

I'm using Django 3.1, which has caused some url issues for me elsewhere. So in admin.py I tried replacing:

`from django.conf.urls import url`

with

`from django.urls import re_path`

and

```
    def get_urls(self):
        return [
            url(r'^(.+)/locking/$', self.admin_site.admin_view(self.locking)),
        ] + super().get_urls()
```
with
```
    def get_urls(self):
        return [
            re_path(r'^(.+)/locking/$', self.admin_site.admin_view(self.locking)),
        ] + super().get_urls()
```
That didn't provide a fix, so I'm back to thinking I don't understand how to use the mixin.

### PR DESCRIPTION
**Pull request by jhrdt**
Fix for issue #4.

The main cause of the malfunction is a missing jQuery init and a removed function (unload()) from jQuery 3.

### CODE REVIEW

File: admin_locking/static/admin_locking/admin_locking.js [hunk 1]
```diff
@@ -11,6 +11,13 @@
     // clear lock when clicking "save"
     $('div.submit-row input[type=submit]').click(clear_lock);
     $(window).unload(clear_lock);
```
Comment (MRigal) [inline admin_locking/static/admin_locking/admin_locking.js]: Isn't the try/catch block aimed to replace this line?

**Review by MRigal — CHANGES_REQUESTED** [review]

File: admin_locking/static/admin_locking/admin_locking.js (hunk 1 above)
Comment (jhrdt) [reply admin_locking/static/admin_locking/admin_locking.js]: Fixed it.

File: admin_locking/admin.py [hunk 2]
```diff
@@ -51,4 +51,7 @@ def get_urls(self):
         ] + super().get_urls()
 
     class Media:
-        js = ('admin_locking/admin_locking.js',)
+        js = (
+            'admin/js/jquery.init.js',
+            'admin_locking/admin_locking.js',
```
Comment (MRigal) [inline admin_locking/admin.py]: Have you tried this version yourself? With Django 3.2, I get the following `jQuery is not a function`, issued from some code coming from jquery.init.js. One option is to add also jquery here, but this starts to be really noisy for a small JS function.

From what I've seen on the net, the best might be actually to drop any jquery reference here, but ensure it is wrapped to be executed only after jquery is loaded, as in https://stackoverflow.com/a/59057562/1504046

File: admin_locking/static/admin_locking/admin_locking.js (hunk 1 above)
Comment (MRigal) [reply admin_locking/static/admin_locking/admin_locking.js]: Thanks!

File: admin_locking/admin.py (hunk 2 above)
Comment (jhrdt) [reply admin_locking/admin.py]: My test setup has no issue.

```
$ pip freeze
Django==3.2.8
django-admin-locking==0.8.3
```

```
$ python3 -V
Python 3.8.10
```

Comment (jhrdt) [reply admin_locking/admin.py]: Hej @MRigal,

could you give more details on your setup? System & Browser? I would like to reproduce your error message. Does it occur while running in debug mode?

Comment (jhrdt) [reply admin_locking/admin.py]: Hi @MRigal, I added the suggest change. Please check 0eb8d0d.

Comment (MRigal) [reply admin_locking/admin.py]: Hi @jhrdt works like a charm now, Thanks a lot!

**Review by MRigal — APPROVED** [review]
Looks great, thanks for the fix! Would you like to add yourself to the CONTRIBUTOR file? Else I may just merge the PR and release a new version right after

**Comment by jhrdt** [comment]
Hi @MRigal,

you can do the merge right away.

**Comment by MRigal** [comment]
v0.9.0 was uploaded to PyPI. Thanks again!

**Pull request merged**